src/include/cpu/rpp_cpu_common.hpp

```diff
@@ -5033,8 +5033,8 @@ inline void compute_generic_bilinear_srclocs_1c_avx(__m256 &pSrcY, __m256 &pSrcX
 inline void compute_generic_bilinear_srclocs_3c_avx(__m256 &pSrcY, __m256 &pSrcX, RpptBilinearNbhoodLocsVecLen8 &srcLocs, __m256 *pBilinearCoeffs, __m256 &pSrcStrideH, __m256i *pxSrcStridesCHW, Rpp32s srcChannels, __m256 *pRoiLTRB, bool isSrcPKD3 = false)
 {
     __m256 pWeightParams[4], pSrcBilinearLTyx[4];
-    pSrcBilinearLTyx[0] = _mm256_floor_ps(pSrcY);                               // srcLT->y = (Rpp32s) srcY;
-    pSrcBilinearLTyx[1] = _mm256_floor_ps(pSrcX);                               // srcLT->x = (Rpp32s) srcX;
+    pSrcBilinearLTyx[0] = _mm256_floor_ps(pSrcY);     // srcLT->y = (Rpp32s) srcY;
```

Comment:
Please revert this change

Reply:
Yes, reverted this in my latest commits, however this isn't getting reflected

Reply:
It was added as part of "[https://github.com/r-abishek/rpp/pull/239/commits/1bcb8dc3e0c3120a82ed09778f9c3f2171ee4369](https://github.com/r-abishek/rpp/pull/239/commits/1bcb8dc3e0c3120a82ed09778f9c3f2171ee4369)"
https://github.com/r-abishek/rpp/blob/1bcb8dc3e0c3120a82ed09778f9c3f2171ee4369/src/include/cpu/rpp_cpu_common.hpp#L5006

Reply:
It was just changed for compute_generic_bilinear_srclocs_1c_avx 
I commented for compute_generic_bilinear_srclocs_3c_avx

Please check the link to the code added in previous comment before replying

Reply:
Oh yeah, fixed this.